### ISSUE TITLE
alltube - nie działą openload.co

### ISSUE THREAD

**Issue by moozgoos** [podpis/kodi#79]
Od jakiegoś czasu nie można otwierać linków openload.co . Mimo, że urządzenie jest sparowane przez olpair.com , ciągle wywala OpenLoad Stream Authorization. Ktoś coś?

**Comment by xxcriticxx** [comment]
@mbebe spojrzysz

@moozgoos jak tak samo mam w iinych wtyczkach. naciskasz X na parowanie openload i film sie zalancza.

**Comment by moozgoos** [comment]
W tym przypadku nie pomaga. Błąd urlresolver: No Openload Authorization.

**Comment by xxcriticxx** [comment]
@moozgoos podaj link do testow z alltube.  mozesz sprawdzic czy masz tak samo na serialosy24.pl.

**Comment by moozgoos** [comment]
Tak, na serialosy jest to samo. Próbuję np obejrzeć Breaking Bad S5E16. W większości pojawia się komunikat o czekaniu na sparowanie urządzenia. W niektórych pozycjach pojawia się błąd, że nie znaleziono linku (w tych przypadkach pewnie rzeczywiście link jest uszkodzony).

**Comment by xxcriticxx** [comment]
@moozgoos tylko na serialosach mozna zaknac X i film/serial sie odpala

**Comment by koladka** [comment]
vpn i po sprawie;)

**Comment by moozgoos** [comment]
VPN na USA np?

**Comment by xxcriticxx** [comment]
ja jade bez VPN ale mam USA ip

**Comment by moozgoos** [comment]
Pewnie to nie miejsce na to, ale czy ktoś może polecić jakiś VPN?

**Comment by xxcriticxx** [comment]
@moozgoos tego jest pelno stym tobie nie bedzie dzialal real debrid

**Comment by koladka** [comment]
nord vpn ;)

czw., 25.10.2018, 15:46 użytkownik Piotr Jaszczyk <notifications@github.com>
napisał:

**Comment by mbebe** [comment]
na czystym kodim, bez zadnych buildow wszystko jest ok.
nie macie przypadkiem autoparowania z cherry lub tym podobne?

**Comment by xxcriticxx** [comment]
@mbebe ja mam dodane repo cherry zeby zamontowac fanfilm i nic wiecej tam nie uzywam

**Comment by notoco** [comment]
@mbebe - na resolverze z repo cherry nie ma autoparowania z openload.

**Comment by xxcriticxx** [comment]
@mbebe tylko w ver vip bylo wczoraj mowione

**Comment by mbebe** [comment]
@notoco - u ciebie to samo sie dzieje z openload?

**Comment by notoco** [comment]
Tak. Na tym normalnym nie ma tego.
teraz sprawdziłem na moim kodi gdzie mam 2 repozytoria - cherry i mbebe. Parowanie z openload działa i odtwarza bez problemowo. Resolver od cherry (bez vip), wtyczki alltube i fanfilm

**Comment by notoco** [comment]
@xxcriticxx - sprawdź adblocka na routerze - mi ostatnio filtry dość mocno blokowały i musiałem "odpuścić" bo openload się nie chciał parować.

**Comment by xxcriticxx** [comment]
@notoco jaki domain?

**Comment by xxcriticxx** [comment]
@notoco poczekaj zrobie test wyloncze wszystko

**Comment by notoco** [comment]
Teraz sobie uświadomiłem że na telefonie gdzie sprawdzałem kodi przed chwilą mam blokada.org praktycznie z tymi samymi plikami host co na routerze... i działa bez problemowo

**Comment by xxcriticxx** [comment]
@notoco nie pomoglo ale zrobie wiecej testow zeby byc sure 100%

ale nie mysle ze @moozgoos ma takie sam setup jak ja

**Comment by xxcriticxx** [comment]
zrobie tak jak @mbebe pisal postawie czyste kodi na weekend

**Comment by moozgoos** [comment]
Ja mam czyste kodi, mam tam jakąś wtyczkę filmwebu, poza tym alltube i pare innych z repozytorium podpis. Nie mam nic więcej. Powyłączałem wszystkie antywirusy itd, nadal nie działa.

**Comment by xxcriticxx** [comment]
@moozgoos masz jakis wtyczki typu incursion, exodus, placenta?

**Comment by mbebe** [comment]
w jakich wtyczkach nie dziala ten openload? tylko alltube i serialosy?

**Comment by krecik71** [comment]
hejka 
u mnie na moim wizardzie działa ok na alltube
mam repo.mbebe ,cherry,podpis itd. Resolver od cherry
kodi17.6

**Comment by xxcriticxx** [comment]
@krecik71 tutaj masz error 

`16:24:20.238 T:18446744071639996704 ERROR: ResolveURL: ResolverError Error - From: openload Link: https://openload.co/embed/2E6J0khhk5A: No OpenLoad Authorization`

**Comment by moozgoos** [comment]
Mi nie działa na alltube i serialosy, mam tylko repo podpis i cherry, resolver autorstwa jsergio v 6.3.4 i to wszystko.

**Comment by xxcriticxx** [comment]
@moozgoos uzywasz real-debrid?

**Comment by moozgoos** [comment]
OK zmieniłem połączenie sieciowe (teraz łączę się z komórki) i zaczęło działać, wygląda na to, że problem leży po stronie użytkownika a nie wtyczki.

Jak dowiem się czemu, to dam znać. Pozdro

**Comment by notoco** [comment]
Sprawdziłem w domu na boxie którego profil mam od zawsze... Wszystko działa, openload alltube + resolver od cherry

**Comment by krecik71** [comment]
@xxcriticxx to przed parowaniem dalej jest ok

**Comment by xxcriticxx** [comment]
@notoco ja paruje sie uzywajac real-debrid + resolver chery/resolver jsergio

**Comment by xxcriticxx** [comment]
@mbebe wszystko dziala na nowym kodi 18 tylko ciekawe jak dlugo :)

update alltuba musisz zrobic bo w repo jest tylko 2.5.5a a u podpisa jest 2.5.5b czy cos innego

pytanie miesiaca ma ktos jakis dzialajacy program co kodi wyczyszcze?

**Comment by notoco** [comment]
https://forum.kodi.tv/showthread.php?tid=307919
U podpisa jest alltube 2.5.5a

**Comment by xxcriticxx** [comment]
gdzies byla 2.5.5b bo ja mam taka wersje zamontowana

**Comment by xxcriticxx** [comment]
@notoco https://github.com/podpis/kodi/issues/60

@mbebe w sumie na ver 2.5.5b z usa juz jest pusto

**Comment by FXcodi** [comment]
Alltube 2.5.5b przestało działać?

**Comment by notoco** [comment]
To nie wtyczka, to strona padła.

**Comment by FXcodi** [comment]
Strona u mnie działa na PC.

**Comment by notoco** [comment]
A tekstu na stronie nie widzisz?
![strona glowna alltube - filmy i seriale online](https://user-images.githubusercontent.com/36659110/51200447-c86caf00-18f9-11e9-94c5-205f16dc70d1.png)

**Comment by kashkash18** [comment]
To przeczytaj jeszcze co pisze na stronie 
![](https://i.imgur.com/cb4JijY.jpg)

**Comment by FXcodi** [comment]
Dziwne, bo napisu nie zauważyłem, ale za to odpaliłem pierwszy lepszy serial.
Widocznie trafiłem w odpowiedni moment.
Grunt, że wiadomo o co chodzi.

**Comment by FXcodi** [comment]
Wtyczka alltube.pl 2.5.8 przestała działać.
